Formula/liboping.rb

```diff
@@ -15,6 +16,11 @@ class Liboping < Formula
 
   uses_from_macos "ncurses"
 
+  patch do
+    url "https://github.com/octo/liboping/pull/55/commits/b5c5091d0214dd2c6cc504c02e2f78907abdee00.patch?full_index=1"
```

Comment:
Since it's now merged, can you use https://github.com/octo/liboping/commit/656ea6996d083090cbbae1581287a3b1a5dcfb56 as the URL?